stackable-operator-derive/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
-authors = ["Stackable GmbH <info@stackable.de>"]
-description = "Stackable Operator Framework"
-edition = "2021"
-license = "Apache-2.0"
 name = "stackable-operator-derive"
-version = "0.45.1"
-repository = "https://github.com/stackabletech/operator-rs"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+description.workspace = true
```

Comment:
I feel like the description for the derive macros should be different to the description of `operator-rs`. Maybe "Derive macros for the Stackable Operator Framework"

Reply:
+1, was just sticking to what we had